main/Fluids.F90

```diff
@@ -865,11 +865,6 @@ SUBROUTINE FLUIDS()
           
        ! Call the modern and significantly less satanic version of study
        call write_diagnostics(state, current_time, dt, timestep)
-       ! Work out the domain volume by integrating the water depth function over the surface if using wetting and drying
```

Comment:
Don't object to this being removed as it's a very random thing to be here but it seems orthogonal to the rest of the pull request.  Worth at least mentioning in the summary?